src/test.ts

```diff
@@ -4,11 +4,11 @@ import { configuration } from "./configure"
 
 export type TestFunction = (instance: any) => void
 
-export function getTests(target: Object): TestFunction[] {
-    const tests = Reflect.getMetadata("test-decorator:tests", target)
+export function getTests(target: any): TestFunction[] {
+    const tests = Reflect.getMetadata(`test-decorator:tests`, target)
```

Comment:
why did you change to template string literals?

Reply:
I originally tried to provide inheritance collisions the tests based on the constructor name of the target